contracts/interface/IPriceFeedDispatcher.sol

```diff
@@ -14,6 +13,9 @@ interface IPriceFeedDispatcher is IPriceFeedDispatcherEvent {
     /// @param interval only useful when using Chainlink; UniswapV3PriceFeed has its own fixed interval
     function dispatchPrice(uint256 interval) external;
 
+    /// @notice can only be initialized once by owner
```

Comment:
According to the past experience, we don't need to add the functions only executed by owner in interface.